learning/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -3,47 +3,89 @@
 C#
 ===
```

Comment:
Can you fix all underlines in this page? They must be triggering tons of warnings.